src/Form/Fields/Utils/SharpFormFieldWithUpload.php

```diff
@@ -67,6 +72,25 @@ public function setCropRatio(string $ratio = null, array $croppableFileTypes = n
         return $this;
     }
 
+    /**
+     * @param bool $shouldOptimize
+     * @return static
+     */
+    public function setShouldOptimize($shouldOptimize = false)
```

Comment:
API name normalization: maybe replace with `shouldOptimizeImage($shouldOptimize = true)` (add Image because file uploads is for all files, remove set and default to true since the method name is clear.
This allows to call `SharpUploadfield::make("test")->shouldOptimizeImage()`.